src/test/java/com/zombiecastlerush/building/InventoryTest.java

```diff
@@ -0,0 +1,81 @@
+package com.zombiecastlerush.building;
+
+import com.zombiecastlerush.role.Player;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.management.relation.Role;
+import java.util.*;
+import static org.junit.Assert.*;
+
+public class InventoryTest {
+    Inventory inventory = new Inventory();
+
+    @Before
+    public void setUp() throws Exception {
+        Item item0 = new Item("item 0", "item 0");
+        inventory.addItems(item0);
+    }
+
+    @Test
+    public void getItems_ReturnsListOfCurrentItems() {
```

Comment:
Non-blocking, but I think you could accomplish this by just creating an ArrayList of the item references and comparing it to inventory.getItems().

Reply:
updated